pkgs/by-name/re/resnap/package.nix

```
@@ -0,0 +1,48 @@
+{
+  lib,
+  stdenv,
+  writeShellApplication,
+  ffmpeg,
+  feh,
+  imagemagick_light,
+  lz4,
+  fetchFromGitHub,
+}:
+let
+  name = "reSnap";
+  version = "2.5.2";
+
+  src = stdenv.mkDerivation {
```

I see that you're using `mkDerivation` to patch the script name. However, we should use `stdenvNoCC` since we're not compiling anything.

```suggestion
  src = stdenvNoCC.mkDerivation {
```

---

Oh, super cool, thanks for showing me this!